includes/batch.inc

```diff
@@ -241,8 +246,18 @@ function islandora_paged_content_create_ocr_batch_operation(array $pages, array
   $page = islandora_paged_content_start_batch_operation($pages, $context);
   if ($page) {
     $page = islandora_object_load($page);
-    $context['results']['successful'][] = islandora_paged_content_page_derive_ocr_datastreams($page, $options);
-    islandora_paged_content_end_batch_operation($context, 1, array('Performed OCR on @successful of @total pages.'));
+    $ocr = islandora_paged_content_page_derive_ocr_datastreams($page, $options, FALSE);
+    foreach ($ocr as $dsid => $result) {
+      if ($result !== FALSE) {
```

Comment:
(It will all be `FALSE` if one of them is: https://github.com/Islandora/islandora_ocr/pull/59/files#diff-f20cd9f6981182cb8c80727774d28d4fR181)